srml/democracy/src/lib.rs

```diff
@@ -509,6 +562,28 @@ mod tests {
 		});
 	}
 
+	#[test]
+	fn starting_referendum_before_proposing_proposal_should_not_work() {
+		with_externalities(&mut new_test_ext(), || {
+			System::set_block_number(1);
+			assert_eq!(
+				Democracy::start_referendum(Box::new(set_balance_proposal(2)), VoteThreshold::SuperMajorityApprove),
+				Err("Cannot start referendum for given proposal if the proposal does not exist, unless called directly by council")
```

Comment:
Nitpick: Place the error message in a `const` variable and use this variable in the test and in the implementation.